src/toil_scripts/exome_variant_pipeline/exome_variant_pipeline.py

```diff
@@ -20,7 +20,7 @@
 from toil_scripts.tools.mutation_callers import run_muse
 from toil_scripts.tools.mutation_callers import run_mutect
 from toil_scripts.tools.mutation_callers import run_pindel
-from toil_scripts.tools.preprocessing import run_gatk_preprocessing
+from toil_scripts.tools.preprocessing import run_preprocessing
```

Comment:
I think @fnothaft pointed out that skipping markdupes doesn't make any sense, so we can maybe just replace the existing `run_gatk_preprocessing` with one that includes mark duplicates?